docs/hybrid_applications.md

```diff
@@ -35,39 +35,121 @@ If you are sure that you don't call the native SDK before the React Native SDK t
+class RNComponentPredicate : ComponentPredicate<Fragment> {
+    override fun accept(component: Fragment): Boolean {
+        // Identify and drop react native screen views
+        if (component.javaClass.toString().startsWith("class com.swmansion.rnscreens")) {
+            return false
+        }
+        if (component.javaClass.toString().startsWith("class com.facebook.react")) {
+            return false
+        }
+        return true
+    }
+
+    override fun getViewName(component: Fragment): String? {
+        return null
+    }
+
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+        return true
+    }
+
+    override fun hashCode(): Int {
+        return javaClass.hashCode()
+    }
+}
+
+// Use it in your configuration
+configuration.useViewTrackingStrategy(FragmentViewTrackingStrategy(true, RNComponentPredicate()))
+```
+
+Then use `@datadog/mobile-react-navigation` to track your views.
+
+#### Instrumenting React Native errors, interactions and resources
 
-Initialize first the native SDK, then the React Native SDK to get auto-instrumentation on React Native user interactions, errors, and resources.
+Wrap your React Native app with the `DatadogProvider` component to automatically register React Native RUM errors, interactions and resources:
 
-The native SDK configuration is used, except for `firstPartyHosts` and `resourceTracingSamplingRate`, which are used for tracing integrations for JavaScript network calls between your app and your backend.
-If you want to have this integration both from the native and the React Native sides, make sure you set `firstPartyHosts` and `resourceTracingSamplingRate` to the same values in both configurations.
+```javascript
+const configuration = {
+    trackResources: true,
+    trackErrors: true,
+    trackInteractions: true
+};
+
+const RNApp = props => {
+    useEffect(() => {
+        /**
+         * In here we can put fake values. The only goal of this call
+         * is to empty the buffer of RUM events.
+         */
+        DatadogProvider.initialize({
+            clientToken: 'fake_value',
+            env: 'fake_value',
+            applicationId: 'fake_value'
+        });
+    }, []);
+    const navigationRef = useRef(null);
+
+    return (
+        <DatadogProvider configuration={configuration}>
+            {/* Content of your app goes here */}
+        </DatadogProvider>
+    );
+};
+
+AppRegistry.registerComponent('RNApp', () => RNApp);
+```
+
+To remove duplicated interactions on Android, filter out the React Native interactions on the native side with an `EventMapper`:
+
+```kotlin
+class RNActionEventMapper : EventMapper<ActionEvent> {
+    override fun map(event: ActionEvent): ActionEvent? {
+        var targetClassName = (event.context?.additionalProperties?.get("action.target.classname") as? String)
+        if(targetClassName?.startsWith("com.facebook.react") == true) {
```

Comment:
same: package name can be obfuscated, we need to add the reminder to put in place Proguard rules then if we rely on that.